forc/src/cli/commands/parse_bytecode.rs

```diff
@@ -59,6 +59,14 @@ pub(crate) fn exec(command: Command) -> ForcResult<()> {
                     parsed_raw
                 )
             }
+            Err(fuel_asm::InvalidOpcode) if word_ix == 4 || word_ix == 5 => {
+                let parsed_raw = u32::from_be_bytes([raw[0], raw[1], raw[2], raw[3]]);
+                format!(
+                    "configurables offset {} ({})",
```

Comment:
Should this be referencing "metadata"?

I'm a little confused as to whether metadata is anything other than the configurables offset. If it's only that, and only can be used for that, we should simplify the code by just calling it that everywhere instead of metadata.

Reply:
> Should this be referencing "metadata"?
> 
> I'm a little confused as to whether metadata is anything other than the configurables offset. If it's only that, and only can be used for that, we should simplify the code by just calling it that everywhere instead of metadata.

> we should simplify the code by just calling it that everywhere instead of metadata

Yes, that was my intention. What you've rightly pointed out in the review comments are places I missed making this changes. I'm about to fix that. Thank you.